packages/medusa/src/services/pricing.ts

```diff
@@ -27,6 +39,8 @@ type InjectedDependencies = {
   regionService: RegionService
   priceSelectionStrategy: IPriceSelectionStrategy
   featureFlagRouter: FlagRouter
+  remoteQuery: (...args: any[]) => Promise<any> | null
```

Comment:
question: Do/should we have a more specific type?

Reply:
added it 😄